aws/ci_build

```diff
@@ -125,11 +125,12 @@ def main
     # Notify the ChatClient channels about what happened.
     if status == 0
       message = "<b>#{projects}</b> built." + time_message + log_link
-
       ChatClient.log message, color: 'green'
 
-      ChatClient.message 'server operations', message, color: 'green'
-      ChatClient.message 'server operations', commit_url, color: 'gray', message_format: 'text'
+      if rack_env == 'production'
```

Comment:
:warning: `rack_env` returns a `Symbol`, so this needs to be `rack_env == :production` or it will never be true